manual/source/Infrastructure/Use/InstallDependencies.rst

```diff
@@ -32,14 +32,15 @@ Windows
 The downloadable contribs contain precompiled library binaries (zlib, libbz2) for Windows by the supported compilers.
 The contribs come in 32 bit and 64 bit variants.
 
-* `Download contribs for 32 bit builds <http://ftp.seqan.de/contribs/seqan-contrib-D20130710-x86.zip>`_.
-* `Download contribs for 64 bit builds <http://ftp.seqan.de/contribs/seqan-contrib-D20130710-x64.zip>`_.
+* `Download contribs for 32 bit builds <http://ftp.seqan.de/contribs/seqan-contrib-D20160115-x86.zip>`_.
+* `Download contribs for 64 bit builds <http://ftp.seqan.de/contribs/seqan-contrib-D20160115-x64.zip>`_.
 
 You can install both variants in parallel if you want to do both 32 bit and 64 bit builds.
+Previous contribs packages are available `here <http://ftp.seqan.de/contribs/>`_ and `here <http://svn.mi.fu-berlin.de/seqan-contrib/>`_ you can find the code for building contribs with new VS versions, for example.
```

Comment:
You need to use anonymous hyperlink references using double underscore after `here <http://ftp.seqan.de/contribs/>`_ and `here <http://svn.mi.fu-berlin.de/seqan-contrib/>`_ to avoid duplicate explicit target name warning in readthedocs.